src/test/java/org/jabref/logic/integrity/UTF8CheckerTest.java

```diff
@@ -42,7 +44,7 @@ void fieldAcceptsUTF8() {
     @Test
     void fieldDoesNotAcceptUmlauts() throws UnsupportedEncodingException {
         UTF8Checker checker = new UTF8Checker(Charset.forName("GBK"));
-        String NonUTF8 = new String("你好，这条语句使用GBK字符集".getBytes(), "GBK");
+        String NonUTF8 = new String("你好，这条语句使用GBK字符集".getBytes(StandardCharsets.UTF_8), GBK);
```

Comment:
Sure that this is intended here?

Reply:
Test goes through. JavaDoc on `getBytes` says that default Charset is used. Maybe be platform dependend. Java switched to UTF8 recently.

See the last parameter `GBK` which recodes the bytes to GKB leading to non-utf. Therefore, I think, it works.